Userland/Libraries/LibGfx/BMPLoader.cpp

```
@@ -1486,7 +1468,7 @@ ErrorOr<ImageFrameDescriptor> BMPImageDecoderPlugin::frame(size_t index)
     if (m_context->state == BMPLoadingContext::State::Error)
         return Error::from_string_literal("BMPImageDecoderPlugin: Decoding failed");
 
-    if (m_context->state < BMPLoadingContext::State::PixelDataDecoded && !decode_bmp_pixel_data(*m_context))
+    if (m_context->state < BMPLoadingContext::State::PixelDataDecoded && decode_bmp_pixel_data(*m_context).is_error())
```

Same note here about manual error checking / hiding the underlying error